test/Confluent.Kafka.IntegrationTests/Tests/Consumer_Poll.cs

```diff
@@ -0,0 +1,79 @@
+// Copyright 2016-2017 Confluent Inc.
```

Comment:
You're not the first to interrogate me on that :-)
Sometimes I think this naming convention helps a lot in grouping like things together.
Particularly when names get long (as @tomasdeml  is suggesting above) it can help a lot.
In the past I've reluctantly conceded I'm wrong but in my own projects I still use it all the time without hesitation.